xml/virt_support.xml

```diff
@@ -4,779 +4,274 @@
-    not required, it is strongly recommended to use them. The
-    paravirtualized drivers are available as follows:
-   </para>
-   <variablelist>
-    <varlistentry os="osuse">
-     <term>&productname;</term>
-     <listitem>
-      <para>
-       Included in kernel
-      </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term>&sls; 12 / 12 SP1 / 12 SP2</term>
-     <listitem>
-      <para>
-       Included in kernel
-      </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry os="sles;sled">
-     <term>&sls; 11 / 11 SP1 / 11 SP2 / 11 SP3 / 11 SP4</term>
-     <listitem>
-      <para>
-       Included in kernel
-      </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry os="sles;sled">
-     <term>&sls; 10 SP4</term>
-     <listitem>
-      <para>
-       Included in kernel
-      </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term>RedHat</term>
-     <listitem>
-      <para>
-       Available in RedHat Enterprise Linux 5.4 and newer
-      </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term>Windows</term>
-     <listitem>
-      <para>
-       &suse; has developed virtio-based drivers for Windows, which are
-       available in the Virtual Machine Driver Pack (VMDP).
-       For more information, see
-       <link xlink:href="https://www.suse.com/products/vmdriverpack/"/>.
-      </para>
-     </listitem>
-    </varlistentry>
-   </variablelist>
-  </sect2>
- </sect1>
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="cha-virt-support">
+ <title>Supported Hosts, Guests, and Features</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+  </dm:docmanager>
+ </info>
+ <para os="sles;sled">
+  Supported architectures and virtualization limits for &xen; and &kvm; are
+  outlined in the <link xlink:href="https://www.suse.com/releasenotes/">Release
+  Notes</link>.
+ </para>
  <sect1 xml:id="virt-support-hosts">
-  <title>
-   Supported &vmhost;s for &productname; &productnumber; &vmguest;s
-  </title>
+  <title>Host Environments (Hypervisors)</title>
+
   <para>
    This section lists the support status of &productname; &productnumber;
-   running as a guest on top of various virtualization hosts (<xref
-   linkend="gloss-vt-hypervisor"/>). Both 32-bit and 64-bit versions are
-   supported for the host if available. The support status is defined as
-   follows:
+   running as a guest on top of various virtualization hosts
+   (<xref
+    linkend="gloss-vt-hypervisor"/>).
   </para>
+
   <itemizedlist>
+   <title>The following &suse; host environments are supported:</title>
    <listitem>
     <para>
-     Full support for all &suse; host systems and &productname;
-     &productnumber; &vmguest;s
+     SLES 11 SP4 XEN+KVM
```

Comment:
I question the "XEN+KVM" on the end of the line with no separator.

Since they are **all** under XEN+KVM and nothing else, I suggest that it should go in the heading above.

Reply:
agree